app/src/main/java/at/aau/serg/activities/LobbyActivity.kt

```diff
@@ -115,6 +127,13 @@ class LobbyActivity : AppCompatActivity() {
         lobbyPlayers[playerToRemove].name = ""
         lobbyPlayers[playerToRemove].isVisible = Visibilities.INVISIBLE
         adapter.notifyItemChanged(playerToRemove)
+
+        if (playerCount() < 3) {
+            this.runOnUiThread {
+                val startBtn: Button = findViewById(R.id.btnStartGame)
+                startBtn.isEnabled = false
```

Comment:
I am not sure if we want also that the not lobby admin is able to start the game

Reply:
Nice catch